src/network/dto/protocol-fees-query.dto.ts

```diff
@@ -0,0 +1,46 @@
+/* istanbul ignore file */
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
+import { TxTag, TxTags } from '@polymeshassociation/polymesh-sdk/types';
+import { Transform } from 'class-transformer';
+import { ArrayNotEmpty, IsArray, IsOptional, IsString } from 'class-validator';
+
+import { IsTxTag } from '~/common/decorators/validation';
+import { getTxTags } from '~/common/utils';
+
+const toArray = (value: unknown): string[] | undefined => {
```

Comment:
This seems like a function that should be generic. I guess its fine to leave, but feels like something that should be in `~/common/utils`